### PR TITLE
Fixes bloodbath effects not working on people in crit

### LINKED ISSUE #24920: Mayhem in a bottle while unconscious
Mayhem in a bottle has a bizarre effect on unconscious people - they still get the usual music and the message to "rip and tear" - but other people do not appear to be changed into slaughter demons and the chainsaw does not really appear in unconscious person's hands - instead it simply appears on the floor.

### PR DESCRIPTION
**Pull request by silicons**
Fixes #24920

### CODE REVIEW

File: code/modules/mob/inventory.dm [hunk 1]
```diff
@@ -244,14 +244,14 @@
 						to_chat(usr, "<span class='notice'>Your [inactive_stack.name] stack now contains [inactive_stack.get_amount()] [inactive_stack.singular_name]\s.</span>")
 						return TRUE
 
-	if(put_in_active_hand(I))
+	if(put_in_active_hand(I, TRUE))
```
Comment (ShizCalev) [inline code/modules/mob/inventory.dm]: Did you mean for these to use forced?

Comment (AnturK) [reply code/modules/mob/inventory.dm]: Bump

Comment (kevinz000) [reply code/modules/mob/inventory.dm]: yesWAITFUCK

File: code/modules/mob/inventory.dm [hunk 2]
```diff
@@ -244,14 +244,14 @@
...
-	if(put_in_active_hand(I))
+	if(put_in_active_hand(I, forced))
 		return TRUE
 
 	var/hand = get_empty_held_index_for_side("l")
 	if(!hand)
 		hand =  get_empty_held_index_for_side("r")
 	if(hand)
-		if(put_in_hand(I, hand))
+		if(put_in_hand(I, hand, TRUE))
```
Comment (AnturK) [inline code/modules/mob/inventory.dm]: I'm pretty sure this one supposed to be `forced` too

File: code/game/objects/effects/mines.dm [hunk 3]
```diff
@@ -123,16 +123,16 @@
...
+	var/static/list/pure_red = list(0,0,0,0,0,0,0,0,0,1,0,0)
 
 	spawn(0)
 		new /datum/hallucination/delusion(victim, TRUE, "demon",duration,0)
 
 	var/obj/item/twohanded/required/chainsaw/doomslayer/chainsaw = new(victim.loc)
 	chainsaw.flags_1 |= NODROP_1
 	victim.drop_all_held_items()
-	victim.put_in_hands(chainsaw)
+	victim.put_in_hands(chainsaw, FALSE, null, TRUE)
```
Comment (Cyberboss) [inline code/game/objects/effects/mines.dm]: null is incorrect here, just use named args to set forced

**Review by Cyberboss — CHANGES_REQUESTED** [review]

File: code/modules/flufftext/Hallucination.dm [hunk 4]
```diff
@@ -471,10 +471,13 @@ GLOBAL_LIST_INIT(hallucinations_major, list(
 	var/image/A = null
 	var/kind = force_kind ? force_kind : pick("monkey","corgi","carp","skeleton","demon","zombie")
 	feedback_details += "Type: [kind]"
+	var/list/nearby
+	if(skip_nearby)
+		nearby = get_hearers_in_view(7, target)
```
Comment (optimumtact) [inline code/modules/flufftext/Hallucination.dm]: why did you move this ?

Comment (kevinz000) [reply code/modules/flufftext/Hallucination.dm]: run a laggy proc 1 time instead of for every mob in glob.moblist @optimumtact

Comment (optimumtact) [reply code/modules/flufftext/Hallucination.dm]: oh right, that makes sense
